src/storage/v2/edge_accessor.hpp

```diff
@@ -102,11 +102,11 @@ class EdgeAccessor final {
   bool for_deleted_{false};
 };
 
-}  // namespace storage
+}  // namespace memgraph::storage
 
 namespace std {
 template <>
-struct hash<storage::EdgeAccessor> {
-  size_t operator()(const storage::EdgeAccessor &e) const { return e.Gid().AsUint(); }
+struct hash<memgraph::storage::EdgeAccessor> {
+  size_t operator()(const memgraph::storage::EdgeAccessor &e) const { return e.Gid().AsUint(); }
 };
```

Comment:
It is not something this PR should fix, just a question: doesn't this will result a very poor distribution of edge accessors in hashtables? As the Gid is strictly incremental, the set of edge accessors might get very close hashes. @gitbuda do you know any reasoning behind this?

Reply:
Not sure, but I think nobody considered that (maybe somebody did), but since `gid` is the only data under edge that will exist for sure, doing hash like this is the most straightforward thing.

Reply:
@kostasrim checked out, and it works just fine.